tests/unit/lib/SGSparseMatrix_unittest.cc

```diff
@@ -23,138 +25,203 @@ using namespace Eigen;
 
 TEST(SGSparseMatrix, multiply_float64_int32)
 {
-	const int32_t size=10;
-	const int32_t num_feat=size/2;
-
-	SGSparseMatrix<float64_t> m(size, size);
-	for (index_t i=0; i<size; ++i)
-	{
-		m.sparse_matrix[i]=SGSparseVector<float64_t>(num_feat);
-		for (index_t j=0; j<num_feat; ++j)
-		{
-			SGSparseVectorEntry<float64_t> entry;
-			entry.feat_index=(j+1)*2;
-			entry.entry=0.5;
-			m.sparse_matrix[i].features[j]=entry;
-		}
-	}
-
-	SGVector<int32_t> v(size);
-	v.set_const(2);
-
-	SGVector<float64_t> result=m*v;
-	Map<VectorXd> r(result.vector, result.vlen);
-
-	EXPECT_NEAR(r.norm(), 12.64911064067351809115, 1E-16);
+  const int32_t size=10;
```

Comment:
I think that your editor has messed around with the white spaces characters. Check if the file was initially using tabs or white spaces and stick to them.